content/docs/command-reference/exp/remove.md

```diff
@@ -38,7 +38,11 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
   include all first-parent commits (similar to `git log -n`).
 
 - `-g`, `--git-remote` - Name or URL of the Git remote to remove the experiment
-  from
+  from.
+
+  If `studio.token` config or `DVC_STUDIO_TOKEN` environment variable is set,
```

Comment:
Reading the docs for it, I'm not sure we need to add this or anything to `exp remove`. It feels like more of an implementation detail than anything needed for usage.